doc/nrf/gs_recommended_versions.rst

```diff
@@ -22,6 +22,74 @@ However, there are some Zephyr features that are currently only available on Lin
 
    Before you start setting up the toolchain, install available updates for your operating system.
 
+.. _gs_supported_OS:
+
+Supported operating systems
+***************************
+
+The following table shows the operating systems and their various versions that provide support for the |NCS| tools:
+
+.. list-table::
+   :header-rows: 1
+
+   * - Operating System
+     - x86
+     - x64
+     - ARM64
+   * - `Windows 11`_
+     - Tier 3
+     - Tier 3
+     - Not supported
+   * - `Windows 10`_
+     - Tier 1
+     - Tier 2
+     - Not supported
+   * - `Linux - Ubuntu 22.04 LTS`_
+     - Not supported
+     - Not supported
+     - Not supported
+   * - `Linux - Ubuntu 20.04 LTS`_
+     - Not supported
+     - Tier 1
+     - Not supported
+   * - `macOS 12`_
+     - Not applicable
+     - Tier 2
+     - Tier 2
+   * - `macOS 11`_
+     - Not applicable
+     - Tier 1
+     - Not supported
+   * - `macOS 10.15`_
+     - Not applicable
+     - Tier 3
+     - Not supported
+
+The table uses the following tier definitions to categorize the level of operating system support:
+
+Tier 1
+  The |NCS| tools will always work.
+  The automated build and automated testing ensure that the |NCS| tools build and successfully complete tests after each change.
+
+Tier 2
+  The |NCS| tools will always build.
+  The automated build ensures that the |NCS| tools build successfully after each change.
+  There is no guarantee that a build will work because the automation tests do not always run.
+
+Tier 3
+  The |NCS| tools are supported by design but are not built or tested after each change.
+  Therefore, the application may or may not work.
+
+Not supported
+  The |NCS| tools do not work.
```

Comment:
```suggestion
  The |NCS| tools do not work, but it may be supported in the future.
```
It felt a little weird to separate the sentences, so I think you can combine them like this.